deployments/scripts/generate-config-for-bootstrap-local-node

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+# Helper script to configure a new Penumbra node locally.
+# Useful in automated test suites.
+set -euo pipefail
+
+
+# Set default values for common pd args, nudging them up 100 ports.
+# This is to avoid the standard defaults, so that we can run multiple
+# nodes on localhost at the same time.
+PENUMBRA_PD_GRPC_BIND="${PENUMBRA_PD_GRPC_BIND:-127.0.0.1:8180}"
+PENUMBRA_PD_ABCI_BIND="${PENUMBRA_PD_ABCI_BIND:-127.0.0.1:26758}"
+PENUMBRA_PD_TM_RPC_BIND="${PENUMBRA_PD_TM_RPC_BIND:-127.0.0.1:26757}"
+PENUMBRA_PD_TM_P2P_BIND="${PENUMBRA_PD_TM_P2P_BIND:-127.0.0.1:26756}"
+PENUMBRA_PD_METRICS_BIND="${PENUMBRA_PD_METRICS_BIND:-127.0.0.1:9100}"
```

Comment:
thought for another day, but it seems like we would get a lot of mileage out of putting these sorts of things on unix domain sockets, in a temp directory, so we aren't left doing math about which ports are available for which nodes.

that's totally not a blocker, just noting that thought now.